plugins/deployment_manager/ManualVerify.ts

```diff
@@ -54,14 +55,15 @@ export async function manualVerifyContract(
     settings.optimizer.runs = 1000000;
   }
 
-  let request = toVerifyRequest({
+  const args = await encodeArguments(contractMetadata.abi, sourceName, contractFQN, deployArgs);
```

Comment:
Hmm, I'm a bit confused about this part. Shouldn't we just be encoding the `deployArgs`?

Reply:
Actually nevermind, just saw that this is a `hardhat-etherscan` function that takes these specific args

Reply:
Isn't that what `encodeArguments` does?

Reply:
https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-etherscan/src/ABIEncoder.ts#L10

Reply:
> Actually nevermind, just saw that this is a `hardhat-etherscan` function that takes these specific args

Ah this comment wasn't showing before 👍